### PR TITLE
Avoid to log `job_id` and `elapsedTime`

### PR DESCRIPTION
**Pull request by dgaubert**
Simplified logger and used at top level of analisis creation

### CODE REVIEW

File: lib/analysis-logger.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+module.exports = AnalysisLogger;
+
+AnalysisLogger.prototype.log = function (analysis) {
+    var self = this;
+
+    analysis.getSortedNodes()
+        .forEach(function (node) {
+            if (node.getType() !== 'source') {
+                self.logger.info({
+                    analysis: analysis.getRoot().id(),
```
Comment (rochoa) [inline lib/analysis-logger.js]: analysis.id()

**Comment by rochoa** [comment]
👍

**Comment by dgaubert** [comment]
I've forgotten to indicate when a node is new or not.

File: lib/analysis-logger.js (hunk 1 above)
Comment (dgaubert) [inline lib/analysis-logger.js]: done!

File: lib/service/database.js [hunk 2]
```diff
@@ -336,10 +325,11 @@ DatabaseService.prototype.queueAnalysisOperations = function(analysis, callback)
                     query: updateNodeStatusAtAnalysisCatalogQuery(nodeIds, Node.STATUS.RUNNING)
                 });
                 asyncQueries.push({
+                    id: [analysis.id(), nodeToUpdate.id(), nodeToUpdate.getType()].join(':'),
```
Comment (dgaubert) [inline lib/service/database.js]: For instance:

`8563bc5307ba5f7231bbc35f4bffb4173cd7f693:8563bc5307ba5f7231bbc35f4bffb4173cd7f693:sampling`

**Pull request merged**